src/main/scala/io/iohk/ethereum/network/p2p/messages/PV63.scala

```diff
@@ -38,8 +40,8 @@ object PV63 {
     }
   }
 
-  object Account {
-    implicit val rlpEndDec = new RLPEncoder[Account] with RLPDecoder[Account] {
+  object AccountImplicits {
+    implicit val rlpEncDec = new RLPEncoder[Account] with RLPDecoder[Account] {
```

Comment:
I prefer to keep logic of encoding/decoding domain object to RLPs in `ethereum.network.p2p.messages` package. I am almost sure that we won't need that anywhere else.